src/Confluent.Kafka/GroupInfo.cs

```diff
@@ -62,36 +62,43 @@ public GroupInfo(BrokerMetadata broker, string group, Error error, string state,
             Members = members;
```

Comment:
Seems like this file needs reverting I don't see a reason for these empty lines.

Reply:
good point. sometimes i use double lines to separate out sections, but it's of no value here.

Reply:
@mhowlett , I was thinking, do you want me to research some static analysis tools/features/extensions that you would like to use for this project, perhaps and enforce some project coding style?

Perhaps something like Coverity, Editorconfig, Stylecop?

Reply:
thanks for the offer. however, IMO there wouldn't be enough payoff for this project (I don't feel this is causing any major problems).